.github/workflows/publish-website.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy Website
+on:
+  push:
+    branches: [main]
+    # uncomment this once the website publishes well
+    # tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: coursier/cache-action@v6
+      - name: Publish ${{ github.ref }}
+        run: sbt docs/docusaurusPublishGhpages
+        env:
+          GITHUB_DEPLOY_KEY: ${{ secrets.GITHUB_DEPLOY_KEY }}
```

Comment:
I lack access to this part of the repo's settings in Github, so I need some assistance setting this token up https://scalameta.org/mdoc/docs/docusaurus.html#deploy-key Maybe @SethTisue can do that?

Reply:
I followed the steps, accurately I hope :-)